include/rosdiscover-clang/Ast/Stmt/SymbolicExpr.h

```diff
@@ -11,6 +11,19 @@ class SymbolicExpr : public SymbolicStmt {
   virtual nlohmann::json toJson() const = 0;
   virtual std::string toString() const = 0;
 
+  virtual std::vector<const SymbolicExpr*> getChildren() const {
+    return {};
+  }
+
+  std::vector<const SymbolicExpr*> getTransitiveChildren() const {
```

Comment:
Minor improvement in method name: rename to `descendants`

Reply:
Fixed